options/options.c

```diff
@@ -114,6 +116,10 @@ static const m_option_t mp_vo_opt_list[] = {
     {"ontop-level", OPT_CHOICE(ontop_level, {"window", -1}, {"system", -2},
         {"desktop", -3}), M_RANGE(0, INT_MAX)},
     {"border", OPT_BOOL(border)},
+#if HAVE_WIN32_DESKTOP && __MINGW64_VERSION_MAJOR >= 12
+    {"backdrop-type", OPT_CHOICE(backdrop_type, {"auto", 0}, {"none", 1},
+        {"mica", 2}, {"acrylic", 3}, {"mica-alt", 4})},
+#endif
```

Comment:
Putting this in the `HAVE_WIN32_DESKTOP` guard below should be sufficient.

Reply:
Wouldn't that still show the users the option even when the mingw version doesn't have the actual headers/code to control the stuff?

Reply:
Sure but you can just make it a no-op. Honestly. Guarding options at all is kind of questionable because it breaks cross-platform config compatibility. You'll notice we're not exactly consistent about this (some platform-specific options are guarded; some aren't).

Reply:
Yeah, I didn't like the idea of guarding it either, especially when it's just due to the mingw version being outdated on the github runner.. i'll put it between HAVE_WIN32_DESKTOP  and force push it